tests/cmdline/commands/test_node.py

```diff
@@ -625,28 +626,30 @@ def test_basics(self):
         """
         from aiida.common.exceptions import NotExistent
 
+        node_delete = VERBOSITY()(cmd_node.node_delete)
```

Comment:
Why does this need to be added here? With the current setup, the subcommands won't have the verbosity option when calling the commands directly through the `cli_runner`? Instead of hotfixing it like this, I think we would need to find a more rigorous global solution

Reply:
> Why does this need to be added here? With the current setup, the subcommands won't have the verbosity option when calling the commands directly through the `cli_runner`?

Correct.

> Instead of hotfixing it like this, I think we would need to find a more rigorous global solution

Happy to hear your solution.
The thing is that the verdi group stuff does not seem to be executed by the cli runner - either you need to force it to run that or you need to find another way to add the option (logically, however, I think the group is the right place to add it).